src/module.ts

```diff
@@ -302,14 +298,27 @@ async function processCollectionItems(nuxt: Nuxt, collections: ResolvedCollectio
       }
       // Sort by file name to ensure consistent order
       list.sort((a, b) => String(a[0]).localeCompare(String(b[0])))
-      collectionDump[collection.name]!.push(...list.flatMap(([, sql]) => sql!))
+      const insertList = list.flatMap(([, sql]) => sql!)
 
-      collectionChecksum[collection.name] = hash(collectionDump[collection.name])
+      collectionChecksum[collection.name] = hash(insertList)
```

Comment:
With this change, the table definition is not used for hash generation. For example, if I change the default value of a number column in the table, we will still use the old table because the hash will not change.